### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by manuwelakanade-GSLAB**
v2.6 additional release notes

### CODE REVIEW

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Can you please review this new request with additional release notes as well. Thank you.

File: docs/release-notes.rst [hunk 1]
```diff
@@ -46,6 +46,32 @@ The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bri
 
 ------
 
+**Hostname vCMP HA peer login during unassign VLANs does not update the login color to Green/Yellow in the side menu**
+
+On a vCMP Host, if a user clicks the **L2-L3 Network Management > vCMP Guest** and selects a vCMP Guest and moves a few VLANs from **Selected** menu to **Available** menu and clicks **Submit**, F5 ACI ServiceCenter logs into the vCMP Guest if it is not already logged in. In this case, the side menu does not show the Green/Yellow color indicator correctly.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:51]: ...if a user clicks **L2-L3 Network Management > vCMP Guest**, selects a vCMP Guest, moves a few VLANs from the **Selected** menu to the **Available** menu, and then clicks **Submit**,

File: docs/release-notes.rst [hunk 2]
```diff
@@ -46,6 +46,32 @@ The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bri
...
 
+**Hostname vCMP HA peer login during unassign VLANs does not update the login color to Green/Yellow in the side menu**
+
+On a vCMP Host, if a user clicks the **L2-L3 Network Management > vCMP Guest** and selects a vCMP Guest and moves a few VLANs from **Selected** menu to **Available** menu and clicks **Submit**, F5 ACI ServiceCenter logs into the vCMP Guest if it is not already logged in. In this case, the side menu does not show the Green/Yellow color indicator correctly. 
+
+**Workaround:** Click the side menu **Refresh BIG-IP List** icon to update the login status of the vCMP Guest.
+
+------
+
+**If a vCMP Guest has been logged in using hostname, vCMP Guest auto-login during VLAN unassignment may result in an error**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:57]: ...logged in using a hostname..

File: docs/release-notes.rst [hunk 3]
```diff
@@ -46,6 +46,32 @@ The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bri
...
+
+The error message observed is: "<IP_Address> is already added as <Hostname>. To add <IP_Address>, delete BIG-IP device <Hostname> and retry."
+
+The steps that may lead to this error are:
+
+- Login to a vCMP Guest using hostname.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:63]: ... a hostname

File: docs/release-notes.rst [hunk 4]
```diff
@@ -46,6 +46,32 @@ The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bri
...
+
+- Login to the corresponding vCMP Host and click **L2-L3 Network Management > vCMP Guest tab**.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:65]: the **L2-L3 Network Management > vCMP Guest** tab.

File: docs/release-notes.rst [hunk 5]
```diff
@@ -97,6 +123,14 @@ If your AS3 declaration contains “optimisticLockKey” mentioned explicitly, t
 
 -------
 
+**Application services declaration delete of scaled config may result in an error 'HTTPError' object has no attribute 'message'**
+
+If an AS3 declaration with scale config is deleted using **L4-L7 App Services > View AS3 Declaration > Delete** button, the F5 ACI ServiceCenter may display an error  'HTTPError' object has no attribute 'message' instead of the actual error that BIG-IP responds with which is '503 Server Error'.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:128]: using the **L4-L7 App Services > View AS3 Declaration > Delete** button, 

the F5 ACI ServiceCenter may display an error stating the 'HTTPError' object has no attribute 'message'

that BIG-IP responds with, which is '503 Server Error'.

File: docs/release-notes.rst [hunk 6]
```diff
@@ -97,6 +123,14 @@ If your AS3 declaration contains “optimisticLockKey” mentioned explicitly, t
 
 -------
 
+**Application services declaration delete of scaled config may result in an error 'HTTPError' object has no attribute 'message'**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:126]: ...an error stating the 'HTTPError' object...

File: docs/release-notes.rst [hunk 7]
```diff
@@ -97,6 +123,14 @@ If your AS3 declaration contains “optimisticLockKey” mentioned explicitly, t
...
+
+If an AS3 declaration with scale config is deleted using **L4-L7 App Services > View AS3 Declaration > Delete** button, the F5 ACI ServiceCenter may display an error  'HTTPError' object has no attribute 'message' instead of the actual error that BIG-IP responds with which is '503 Server Error'. 
+
+**Workaround**: This 503 error occurs when BIG-IP is in error state or is already in the process of configuring a previous AS3 declaration. Once BIG-IP is in steady state and UI is accessible, the delete operation can be retried to get a success response for delete declaration.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:130]: get a successful response for deleting the declaration.

**Review by JosephPJordan — CHANGES_REQUESTED** [review]
@manuwelakanade-GSLAB, a couple minor edits.  Thanks!

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : I have made changes as per your review. Thanks.

**Review by JosephPJordan — APPROVED** [review]

**Pull request merged**